9.0.2 Release

su-sws-jira opened the pull request:
# DO NOT DELETE

su-sws-jira committed:
9.0.2

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr850-urrtthe3zuzei5quhshjwsarjchjvbb5.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736

Visual Diffs:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736/visualdiffs

pookmish committed:
Use new branch name in automated dependency updates

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr850-urrtthe3zuzei5quhshjwsarjchjvbb5.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736

Visual Diffs:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736/visualdiffs

pookmish committed:
Disable dblog on code deployment

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr850-urrtthe3zuzei5quhshjwsarjchjvbb5.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736

Visual Diffs:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736/visualdiffs

pookmish committed:
HSD8-1038 fixed missing "askQuestion" method

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr850-urrtthe3zuzei5quhshjwsarjchjvbb5.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736

Visual Diffs:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736/visualdiffs

pookmish committed:
patched path_redirect_import for D9 compatability

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr850-urrtthe3zuzei5quhshjwsarjchjvbb5.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736

Visual Diffs:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736/visualdiffs

pookmish committed:
HSD8-1033 Collection paragraph type. (#852)

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr850-urrtthe3zuzei5quhshjwsarjchjvbb5.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736

Visual Diffs:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736/visualdiffs

pookmish committed:
HSD8-1034 New "News Style" layout option

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr850-urrtthe3zuzei5quhshjwsarjchjvbb5.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736

Visual Diffs:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736/visualdiffs

pookmish committed:
Reverted dblog uninstall since mrc needs it.

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr850-urrtthe3zuzei5quhshjwsarjchjvbb5.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736

Visual Diffs:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736/visualdiffs

Commit:
STN-675: WEST Gradient Hero --> configuration (#841)

* feat (STN-675): West Gradient Hero configuration

* feat: add configuration files

* feat: update configuration so that captions/credits display on gradient hero images

* feat: update paragraph and row component config files so that paragraph types are listed alphabetically

* feat: update codeception test for the gradient hero paragraph type

* feat: update configuration files to hide the gradient hero from components by default

* feat: update gradient hero paragraph entity view display configuration so that the hero title displays as an h2 tag

* feat: update components configs to match the release and sprint branches before the addition of the gradient hero

* feat: Gradient hero configuration updates

- Add smart trim to body
- Update image styles to the correct setting which matches the hero image overlay component

* feat: address review feedback

- require variant conditional field to match right variant to prevernt adding a variant class to the default hero
- Update the hero alignment configuration to make the alignment a required field. This elminates -None- as an option for users in the alignment dropdown.

* feat: update number of components to be tested

* feat: fix syntax in failing test

* feat: update number of elements tested

* feat: update gradient hero form display to show text area character limit feedback

Commit:
STN-675: Add Gradient Hero Styles 🎨 (#846)

* feat (STN-675): add styles to the gradient hero paragraph type

* feat: add compiled CSS & JS

Commit:
STN-674: Callout Box Component (#853)

* feat(STN-674): Callout Box

 - chore: config for callout box
 - chore: callout box template
 - chore: callout box styling
 - chore: update codeception test
 - chore: fix hs_flexible_components.yml
 - chore: remove hs-callout from row component
 - update: callout-box z-index

* chore: compiled css

pookmish committed:
HSD8-1045 Add "addtocal" module for add to calendar links (#854)

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr850-urrtthe3zuzei5quhshjwsarjchjvbb5.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736

Visual Diffs:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736/visualdiffs

pookmish committed:
update hook

pookmish committed:
Sparkbox Sprint 38 (#851)

* STN-675: WEST Gradient Hero --> configuration (#841)

* feat (STN-675): West Gradient Hero configuration

* feat: add configuration files

* feat: update configuration so that captions/credits display on gradient hero images

* feat: update paragraph and row component config files so that paragraph types are listed alphabetically

* feat: update codeception test for the gradient hero paragraph type

* feat: update configuration files to hide the gradient hero from components by default

* feat: update gradient hero paragraph entity view display configuration so that the hero title displays as an h2 tag

* feat: update components configs to match the release and sprint branches before the addition of the gradient hero

* feat: Gradient hero configuration updates

- Add smart trim to body
- Update image styles to the correct setting which matches the hero image overlay component

* feat: address review feedback

- require variant conditional field to match right variant to prevernt adding a variant class to the default hero
- Update the hero alignment configuration to make the alignment a required field. This elminates -None- as an option for users in the alignment dropdown.

* feat: update number of components to be tested

* feat: fix syntax in failing test

* feat: update number of elements tested

* feat: update gradient hero form display to show text area character limit feedback

* STN-675: Add Gradient Hero Styles 🎨 (#846)

* feat (STN-675): add styles to the gradient hero paragraph type

* feat: add compiled CSS & JS

* STN-674: Callout Box Component (#853)

* feat(STN-674): Callout Box

 - chore: config for callout box
 - chore: callout box template
 - chore: callout box styling
 - chore: update codeception test
 - chore: fix hs_flexible_components.yml
 - chore: remove hs-callout from row component
 - update: callout-box z-index

* chore: compiled css

* update hook

Co-authored-by: KelbelR <kelbel@heysparkbox.com>
Co-authored-by: Mike Decker <pookmish@gmail.com>

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr850-urrtthe3zuzei5quhshjwsarjchjvbb5.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736

Visual Diffs:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736/visualdiffs

pookmish committed:
Label changes to fields and uninstall printfriendly

pookmish committed:
HSD8-1041 HSD8-1042 HSD8-1043 Provision 3 sites

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr850-urrtthe3zuzei5quhshjwsarjchjvbb5.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736

Visual Diffs:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736/visualdiffs

pookmish committed:
Delete the old view to be replaced by the newly named view

pookmish committed:
revert spatie/calendar-links back to 1.5.0

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr850-urrtthe3zuzei5quhshjwsarjchjvbb5.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736

Visual Diffs:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736/visualdiffs

pookmish committed:
HSD8-1033 QA fixes for collections

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr850-urrtthe3zuzei5quhshjwsarjchjvbb5.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736

Visual Diffs:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736/visualdiffs

pookmish committed:
added and enabled migrate_source_csv module

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr850-urrtthe3zuzei5quhshjwsarjchjvbb5.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736

Visual Diffs:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736/visualdiffs

pookmish committed:
cc fixup

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr850-urrtthe3zuzei5quhshjwsarjchjvbb5.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736

Visual Diffs:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736/visualdiffs

pookmish committed:
Fixed update hook and added some news layout styles

pookmish committed:
added missing docblock

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr850-urrtthe3zuzei5quhshjwsarjchjvbb5.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736

Visual Diffs:
- https://dashboard.tugboat.qa/6076649829bd31a21f347736/visualdiffs
